plugins/pipedream/api_key.go

```diff
@@ -0,0 +1,100 @@
+package pipedream
+
+import (
+	"context"
+
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func APIKey() schema.CredentialType {
+	return schema.CredentialType{
+		Name:          credname.APIKey,
+		DocsURL:       sdk.URL("https://pipedream.com/docs/api_key"),                 // TODO: Replace with actual URL
+		ManagementURL: sdk.URL("https://console.pipedream.com/user/security/tokens"), // TODO: Replace with actual URL
+		Fields: []schema.CredentialField{
+			{
+				Name:                fieldname.APIKey,
+				MarkdownDescription: "API Key used to authenticate to Pipedream.",
+				Secret:              true,
+				Composition: &schema.ValueComposition{
+					Length: 32,
+					Charset: schema.Charset{
+						Lowercase: true,
+						Digits:    true,
+					},
+				},
+			},
+			{
+				Name:                fieldname.OrgID,
+				MarkdownDescription: "OrgId for the Pipedream organization.",
+				Secret:              true,
+				Composition: &schema.ValueComposition{
+					Length: 9,
+					Charset: schema.Charset{
+						Lowercase: true,
+						Uppercase: true,
+						Digits:    true,
+						Symbols:   true,
+					},
+				},
+			},
+		},
+		DefaultProvisioner: provision.TempFile(
+			pipedreamConfig,
+			provision.AtFixedPath("~/.config/pipedream/config"),
+		),
+		Importer: importer.TryAll(
+			TryPipedreamConfigFile(),
+		)}
+}
+
+func TryPipedreamConfigFile() sdk.Importer {
```

Comment:
Pipedream supports the concepts of profiles:

https://pipedream.com/docs/cli/login/#using-the-cli-to-manage-multiple-accounts

Could we import credentials from all the profiles? Fastly shell plugin has an example of it using [Namehint](https://pkg.go.dev/github.com/1Password/shell-plugins/sdk#ImportCandidate.NameHint):

https://github.com/1Password/shell-plugins/blob/f57be66f86be90cf47450b1b0c4226c8c669c2a5/plugins/fastly/api_token.go#L51-L68